src/components/Events/Events.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Typography } from "@mui/material";
+import ClientDetails from "./ClientDetails";
+import EncryptionDetails from "./EncryptionDetails";
+import GrantTypeManagement from "./GrantTypeManagement";
+import KeyManagement from "./KeyManagement";
+import DeleteCredentials from "./DeleteCredentials";
+import AppDomains from "./AppDomains";
+
+const Events =() =>{
```

Comment:
rename to events container,

move other components to a separate folder at same level that denotes EventsComponents and then have the components there